magento/pwa-studio
# [Doc] Refactor tutorials section

## Description

This PR reorganizes the tutorials section to make it more lean and streamlined.
Out of scope topics have been removed(archived) and nav items have been re-categorized.

## Related Issue





Closes PWA-1190

## Acceptance

@davemacaulay 
@supernova-at 

### Verification Stakeholders

@davemacaulay 
@supernova-at 

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Run the HTML preview server: `yarn develop`
3. Click on the `Tutorials` card
4. Verify you are taken to a Getting Started page in the tutorials section
5. Verify tutorials section navigation makes sense

## Screenshots / Screen Captures (if appropriate)

![Screen Shot 2020-12-14 at 11 48 22 AM](https://user-images.githubusercontent.com/4692281/102116422-8e1c2d00-3e02-11eb-83b8-f5180b070e9e.png)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
